src/audio/module_adapter/module/generic.c

```diff
@@ -198,7 +198,7 @@ int module_prepare(struct processing_module *mod,
 		   struct sof_source **sources, int num_of_sources,
 		   struct sof_sink **sinks, int num_of_sinks)
 {
-	int ret;
+	int ret = 0;
```

Comment:
I made without knowing about this separate PR #8605. You can leave this fix out from DRC PR. I will add credits of patch to you since you did this first.